src/workers/cdc.ts

```diff
@@ -195,6 +196,14 @@ const onUserChange = async (
         }
       }
     }
+    if (data.payload.before.username !== data.payload.after.username) {
```

Comment:
To make sure we submit this event only when the info is confirmed. Otherwise on every registration we will dispatch this event

```suggestion
    if (data.payload.before.infoConfirmed && data.payload.before.username !== data.payload.after.username) {
```